waspc/packages/deploy/src/providers/railway/commands/setup/setup.ts

```diff
@@ -122,14 +122,42 @@ async function setupRailwayProjectForDirectory({
 
 async function setupDb({
   cmdOptions: options,
+  dbServiceName,
 }: DeploymentInstructions<SetupCmdOptions>): Promise<void> {
   waspSays("Setting up database");
 
   const railwayCli = createCommandWithCwd(
     options.railwayExe,
     options.waspProjectDir,
   );
-  await railwayCli(["add", "-d", "postgres"]);
+
+  if (options.dbImage) {
+    waspSays(`Using custom database image: ${options.dbImage}`);
+    // When using a custom database image, the automatic variables that Railway sets up for the
+    // default Postgres template are not available.
```

Comment:
Are not available?
As in it does not set them?

```suggestion
    // When using a custom database image, 
    // Railway does not set up the default Postgres environment variables.
```

Or maybe just be general then?

```suggestion
    // When using a custom database image, 
    // we must set up the environment variables ourselves.
```